client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.js

```diff
@@ -92,49 +113,80 @@ FileNavCell.propTypes = {
   scheme: PropTypes.string.isRequired,
   href: PropTypes.string.isRequired,
   isPublic: PropTypes.bool,
-  length: PropTypes.number.isRequired
+  length: PropTypes.number.isRequired,
+  trashInProgress: PropTypes.bool.isRequired
 };
 FileNavCell.defaultProps = {
   isPublic: false
 };
 
 export const FileLengthCell = ({ cell }) => {
-  const bytes = cell.value;
+  const bytes = cell.value[0];
+  const trashInProgress = cell.value[1];
 
-  return <span>{formatSize(bytes)}</span>;
+  return trashInProgress ? (
+    <div styleName="disabled">{formatSize(bytes)}</div>
```

Comment:
It's going to be difficult to maintain this going forward if every table cell has to individually account for the trash status (what if we decide on other operations that disable a row? Then every cell component would have to be modified). Is it possible to set a style on the row itself, through the `DataFilesTableRow` component, that disables all of its children? 

You can apply a mapping to the `data` prop `DataFilesTable` that sets an enabled/disabled status on each file, then use that so you don't have to deal with this at the cell level.